docs/source/pages/using.rst

```diff
@@ -168,3 +168,18 @@ Thanks to Datalog’s expressive language model, it’s easy to add exception ru
 requirement. For example, `the Mysql Connector/J <https://slsa.dev/spec/v0.1/requirements#build-service>`_ dependency in
 the Micronaut MuShop project does not pass the ``build_service`` check, but can be manually investigated and exempted if trusted. Overall, policies expressed in Datalog can be
 enforced by Macaron as part of your CI/CD pipeline to detect regressions or unexpected behavior.
+
+---------------------------
+Analyzing more dependencies
+---------------------------
+
+In some cases the dependencies that Macaron discovers lack a direct connection to a repository for it to analyze. To improve results in these instances, the Repository Finding feature can be enabled. This feature makes use of a dependency's identifying information and the matching information that can be found within package managing repositories located on the Internet.
+
+.. note:: The Repository Finding feature currently only works for Java projects via SCM meta data found within artifact POM files.
+
+Enabling this feature requires modification of the ``defaults.ini`` config file located at: ``<macaron_root_directory>/src/config/``. Under the ``repofinder.java`` header, four options of importance exist: ``find_repos``, ``artifact_repositories``, ``repo_pom_paths``, ``find_parents``. These options behave as follows:
```

Comment:
I wonder if we want the user to modify the source code of Macaron in ``<macaron_root_directory>/src/config/``?. I think because this section give instructions on how to run Macaron as a Docker image, we instructs the user to provide a custom `defaults.ini` with the relevant field and run with:
```
./run_macaron.sh -dp <path_to_custom_defaults> analyze ...
```